diff -urN w32api-3.1.orig/lib/directx/Makefile.in w32api-3.1/lib/directx/Makefile.in
--- w32api-3.1.orig/lib/directx/Makefile.in	2004-09-05 17:16:16.000000000 +0200
+++ w32api-3.1/lib/directx/Makefile.in	2004-09-22 23:25:53.137486200 +0200
@@ -72,7 +72,7 @@
 EXTRA_INCLUDES = -I$(srcdir)/../../mingw/include
 endif
 
-INCLUDES = -I$(srcdir)/../include -I$(srcdir)/../include/directx $(EXTRA_INCLUDES)
+INCLUDES = -I$(srcdir)/../../include -I$(srcdir)/../include/directx $(EXTRA_INCLUDES)
 
 CFLAGS = @CFLAGS@
 ALL_CFLAGS = $(CFLAGS) $(INCLUDES)
